README.md

```diff
@@ -6,6 +6,7 @@
 1. Move `.env-sample` to `.env` and replace `TOKEN` (ask @benheller)
 1. `nvm install` <-- select node version
 1. `yarn install` <-- install deps
+1.  E.g. `sudo service mongod start` if running mongo locally
```

Comment:
If these changes aren't actually part of this PR (but rather #26), then please rebase this PR on top that branch so that the diff reflects only the changes introduced in this particular PR. This eliminates potential confusing around receiving feedback on the same code in multiple places.

Reply:
this is a pain in the bum